libs/java/server_common/src/main/java/com/yahoo/athenz/common/server/http/HttpDriver.java

```diff
@@ -258,8 +259,7 @@ public String doGet(final String url) throws IOException {
      */
     public String doPost(HttpPost httpPost) throws IOException {
         String url = httpPost.getURI().toString();
-        String query = EntityUtils.toString(httpPost.getEntity());
-        LOGGER.debug("Requesting from {} with query {}", url, query);
+        LOGGER.debug("Requesting from {} with query {}", url, getPostQuery(httpPost));
```

Comment:
we should put this in If (LOGGER.isDebugEnabled()) block otherwise we'll do all the work for the getPostQuery, then pass the argument to the function which will just throw it away since in production we don't have debug enabled (so the getPostQuery call is just wasted).